modules/imgproc/include/opencv2/imgproc/logo.hpp

```diff
@@ -0,0 +1,146 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include <opencv2/core/mat.hpp>
+#include <opencv2/core/types.hpp>
+
+namespace cv {
+
+/** Class for drawing OpenCV logo
+
+OpenCV logo consists of three "C" letters at different angles placed in corners of a triangle.
+*/
+class Logo
+{
+public:
+    //! Location for drawing the logo
+    enum Location
+    {
+        BottomRight,
+        BottomLeft,
+        TopRight,
+        TopLeft
```

Comment:
Not sure that we really need that.
Users can specify image ROI for logo through image (location and size)